src/sentry/static/sentry/app/views/settings/components/forms/formPanel.tsx

```diff
@@ -19,8 +19,8 @@ type Props = {
   fields: FieldObject[];
 
   // TODO(ts): See if this is still in use
-  access?: Scope[];
-  features?: string[];
+  access?: any;
```

Comment:
I think the old types should be correct

Reply:
I've double checked and `Scope[] `should be fine if I cast it, but `features` is not correct. Please see: 
![image](https://user-images.githubusercontent.com/29228205/74938294-09cd6900-53ee-11ea-8fe4-b0eda5b5bdd9.png)

Reply:
if I change features to be Array<string> I will have to update many files..otherwise we will get errors

Reply:
Ah damn, I think `features` is expecting a Set (We use these very inconsistently unfortunately)